static/scss/answers/collapsible-filters/collapsible-filters-templates.scss

```diff
@@ -45,6 +45,20 @@
 .AnswersVerticalGrid.CollapsibleFilters {
   @include CollapsibleFiltersOverrides;
 
+  .Answers-resultsHeader, .Answers-results {
+    @media (min-width: #{$breakpoint-expanded-filters}) {
+      width: math.div($container-width-at-collapsed-filters-breakpoint, $breakpoint-collapsed-filters) * 100%;
```

Comment:
What are your thoughts on making a sass variable for this calculation such as $answers-container-width-percentage and putting it in sdk-filter-overrides.scss? That way we won't have the formula duplicated twice

Reply:
make sense, I will make that change